packages/amplify-graphql-schema-generator/src/datasource-adapter/mysql-datasource-adapter.ts

```diff
@@ -260,4 +277,22 @@ export class MySQLDataSourceAdapter extends DataSourceAdapter {
 
     return result;
   }
+
+  private getEnumValues(value: string): string[] {
+    // RegEx matches strings with quotes 'match' or "match"
+    const regex = /(["'])(?:(?=(\\?))\2.)*?\1/g;
```

Comment:
can you please add a unit test for this method? Thanks

Reply:
This test hits this function.
https://github.com/aws-amplify/amplify-category-api/pull/1207/files#diff-59c5ce236f8d95da1a965de96f1a36dbb57bc5ccfa0b7d73e0d706f44ab92535R119-R131

Reply:
This test invokes this method.
https://github.com/aws-amplify/amplify-category-api/pull/1207/files#diff-59c5ce236f8d95da1a965de96f1a36dbb57bc5ccfa0b7d73e0d706f44ab92535R63-R66